belga-event-queries.md

```diff
@@ -0,0 +1,186 @@
+# Belga Event Query Guide
+
+## Overview
+This guide documents the Production API queries for filtering events according to Belga's requirements:
+
+- Calendar selection
+- Region filtering (Belgium/International)
+- Date range filtering 
+- Text search
+- Coverage type filtering
+
+## Authentication
+All requests require JWT authentication:
+
+```bash
+export PRODAPI="http://localhost:5500/prodapi/v1"
+export JWT_TOKEN="your.jwt.token"
+```
+
+## Event Queries
+
+### 1. Filter by Calendar
+```json
+GET /prodapi/v1/events
+{
+  "query": {
+    "bool": {
+      "filter": [
+        { "terms": { "calendars._id": ["CALENDAR_ID_1", "CALENDAR_ID_2"] } }
+      ]
+    }
+  }
+}
+```
+
+### 2. Filter by Region
```

Comment:
I believe Belga won't be using the `place` field but instead the `location` field.
So in this case it would be searching `location.address.country`.
Example location:
```json
{
    "location": [
        {
            "address": {
                "area": "BRUSSEL",
                "country": "Belgium",
                "city": "BRUSSEL",
                "line": [
                    "Koningsplein, 7A"
                ],
                "locality": "Brussels",
                "state": "Brussels"
            },
            "name": "Grondwettelijk Hof",
            "qcode": "309e332f-6855-4a9e-bbb7-58594f35ffc2",
            "details": [
                ""
            ],
            "formatted_address": "Koningsplein, 7A BRUSSEL Brussels Belgium"
        }
    ]
}
```